git_downloader.py

```diff
@@ -39,15 +39,17 @@ def file_repo_gen(repo_file, g):
 def download_files(args, g, repo_gen):
     file_counter = 0
     for repo in repo_gen:
-        logging.info('Fetching repository: %s (id: %i)' % (repo.full_name, repo.id))
+        
         try:
+            logging.info('Fetching repository: %s (id: %i)' % (repo.full_name, repo.id))
             tree = repo.get_git_tree('master', recursive=True)
             files_to_download = []
             for file in tree.tree:
                 if fnmatch.fnmatch(file.path, args.wildcard):
                     files_to_download.append('https://github.com/%s/raw/master/%s' % (repo.full_name, file.path))
             for file in files_to_download:
                 logging.info('Downloading %s' % file)
+                file = file.replace(" ", "%20")
```

Comment:
python has a module that handle this quoting issues:
- [Python2](https://docs.python.org/2/library/urllib.html#urllib.quote)
- [Python3](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote)

Can you use them instead?